transport/src/main/java/io/netty/channel/embedded/EmbeddedChannel.java

```diff
@@ -1132,6 +1150,17 @@ public Builder handlers(ChannelHandler... handlers) {
             return this;
```

Comment:
```suggestion
            this.handler = null;
            return this;
```

To make sure the array path is used